src/test/java/codeu/controller/ConversationServletTest.java

```diff
@@ -64,19 +65,40 @@ public void setup() {
     conversationServlet.setUserStore(mockUserStore);
   }
 
+
   @Test
   public void testDoGet() throws IOException, ServletException {
```

Comment:
can you be more specific about what the test does

e.g. testDoGet_privateConversation_success, testDoGet_badUser_failure etc.